integ-tests/hadoop/pom.xml

```diff
@@ -21,36 +21,69 @@
   <artifactId>hadoop</artifactId>
   <version>1.0-SNAPSHOT</version>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <maven-compiler.version>3.8.0</maven-compiler.version>
+    <maven-surefire.version>3.0.0-M5</maven-surefire.version>
+    <gson.version>2.9.0</gson.version>
+    <glassfish.version>3.0.0</glassfish.version>
+    <hive.version>3.1.2</hive.version>
+    <slf4j.version>2.0.0-alpha5</slf4j.version>
+    <os-maven-plugin.version>1.7.0</os-maven-plugin.version>
+    <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>2.0.0-alpha5</version>
+      <version>${slf4j.version}</version>
     </dependency>
 
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-jdk14</artifactId>
-      <version>2.0.0-alpha5</version>
+      <version>${slf4j.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-jdbc</artifactId>
+      <version>${hive.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.el</artifactId>
```

Comment:
The license of this dependency is problematic. IIUC it is GPLv2 or CDDC. GPLv2 is a no-go. CDDC requires us to make the source code available (extra work).

Can we remove the dependency?

Reply:
Yes, I removed it.